sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -208,6 +223,6 @@ def test_mean_shift_zero_bandwidth():
     ms_nobinning = MeanShift(bin_seeding=False).fit(X)
     expected_labels = np.array([0, 0, 0, 1, 1, 1, 2, 2])
 
-    assert v_measure_score(ms_binning.labels_, expected_labels) == 1
-    assert v_measure_score(ms_nobinning.labels_, expected_labels) == 1
+    assert_allclose(v_measure_score(ms_binning.labels_, expected_labels), 1)
+    assert_allclose(v_measure_score(ms_nobinning.labels_, expected_labels), 1)
     assert_allclose(ms_binning.cluster_centers_, ms_nobinning.cluster_centers_)
```

Comment:
Since `v_measure_score` returns a single float then I would use a single `assert`. Since this is still a float, I would use `pytest.approx`.

Reply:
It's preferable because we made a custom ``assrt_allclose`` that adapts its rtol based on the dtype of the arrays

Reply:
Yes indeed it's better to use the scikit-learn's `assert_allclose` in `global_dtype` paramatrized tests because it can automatically handled the `rtol` value based on the dtype when using the default argument.

When comparing to zero with an absolute tol, either option are fine, in my opinion.

Note that `pytest.approx` without arguments uses a relative tol of 1e-6 + an absolute tol of 1e-12.

Reply:
Indeed in this case, the `.labels_` attributes are integer dtyped, whatever the `global_dtype` for `X_train`. Therefore using a generic `pytest.approx` value with default tolerance is fine.